cmd/opm/alpha/render-graph/cmd.go

```diff
@@ -60,7 +60,7 @@ $ opm alpha render-graph quay.io/operatorhubio/catalog:latest | \
 			}
 
 			render.Refs = args
-			render.AllowedRefMask = action.RefBundleImage | action.RefDCImage | action.RefDCDir // all non-sqlite
+			render.AllowedRefMask = action.RefBundleImage | action.RefDCImage | action.RefDCDir | action.RefSqliteImage | action.RefSqliteFile
```

Comment:
Probably doesn't make sense to render a graph for a bundle image, right? Since there's no channel there?